index.html

```diff
@@ -9,15 +9,30 @@
 </head>
```

Comment:
- [ ] Kindly add the following meta tag `<meta name="viewport" content="width=device-width, initial-scale=1" />` inside of `<head>`, so when I click on the mobile toggle icon, I can see the mobile version, not the desktop also as shown in the following image.
>![image](https://user-images.githubusercontent.com/53402600/202469121-54d0be80-e648-41b4-ac06-942e72bf9114.png)